onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2843,6 +2843,72 @@ It's an extension of Gelu. It takes the sum of input A and bias input B as the i
         }
       });
 
+  static const char* Trilu_ver1_doc = R"DOC(
+	  Returns the upper or lower triangular part of a 2-D matrix, or batches of 2-D matrices depending on whether attribute upper is true or not.
```

Comment:
More clear description: "Returns the upper or lower triangular part of a 2-D matrix, or batches of 2-D matrices. If setting attribute `upper` to true, the upper triangular matrix is retained. Lower triangular matrix is retained otherwise."

Specify the default behavior.